telegram/inline/inlinequery.py

```diff
@@ -75,14 +75,21 @@ def de_json(cls, data, bot):
 
         return cls(bot=bot, **data)
 
-    def answer(self, *args, **kwargs):
+    def answer(self, *args, auto_pagination=False, **kwargs):
         """Shortcut for::
 
-            bot.answer_inline_query(update.inline_query.id, *args, **kwargs)
+            bot.answer_inline_query(update.inline_query.id,
+                                    *args,
+                                    current_offset = self.offset if auto_pagination else None,
+                                    **kwargs)
```

Comment:
```suggestion
            bot.answer_inline_query(
                update.inline_query.id,
                *args,
                current_offset=self.offset if auto_pagination else None,
                **kwargs,
            )
```

Reply:
I guess that is a matter of taste … And I'd prefer to stick to the style we use in all the other docstrings